static/styles/settings.css

```diff
@@ -860,6 +861,23 @@ input[type="checkbox"] {
     }
 }
 
+#bot_avatar_img {
+    margin: 5px 0 8px;
+}
+
+#add_bot_preview_text,
+.edit_bot_avatar_preview_text {
+    display: none;
+}
+
+.edit_bot_avatar_preview_image,
+#add_bot_preview_image {
+    height: 100px;
+    object-fit: cover;
+    width: 100px;
+    margin: 2px 0 8px;
+}
```

Comment:
I didn't polish this as much as could be, but I at least moved some of this CSS to be scoped inside the form elements they should live in.

A nicer solution would be to make this logic be common `upload_widget` logic -- like we could have have an `.upload_widget_preview_image` class with these properties that is used on both forms, rather than custom CSS like this. I'd appreciate a follow-up PR doing that sort of refactoring.

Reply:
Yes, I totally agree. Making the logic common and refactoring the code into a ```.upload_widget_preview_image``` class would definitely make the code cleaner and easier to maintain. I'll be happy to contribute with the follow-up PR to implement this change.